commit 45e4039c3aea597ede44a264cea322908cdedfe9
Author: Dave Airlie <airlied@redhat.com>
Date:   Sun Feb 20 21:57:32 2011 +0000

    drm/radeon: fix regression with AA resolve checking
    
    Some userspaces can emit a whole packet without disabling AA resolve
    by the looks of it, so we have to deal with them.
    
    Signed-off-by: Dave Airlie <airlied@redhat.com>
    Tested-by: Jorg Otte <jrg.otte@googlemail.com>
    [dannf: adjusted to apply to Debian's 2.6.32]

diff --git a/drivers/gpu/drm/radeon/r100.c b/drivers/gpu/drm/radeon/r100.c
--- a/drivers/gpu/drm/radeon/r100.c	2011-03-24 23:14:56.114453855 -0600
+++ b/drivers/gpu/drm/radeon/r100.c	2011-03-25 00:43:45.419590454 -0600
@@ -3036,7 +3036,7 @@ void r100_cs_track_clear(struct radeon_d
 		track->num_texture = 16;
 		track->maxy = 4096;
 		track->separate_cube = 0;
-		track->aaresolve = true;
+		track->aaresolve = false;
 		track->aa.robj = NULL;
 	}
 
